tests/unit/models/test_component.py

```diff
@@ -27,15 +63,21 @@ def test_component_metadata(instance):
     )
 
 
-@given(st.builds(Component, name=st.from_regex(PERMITTED_NAME_REGEX)))
+# @given(st.builds(Component, name=st.from_regex(PERMITTED_NAME_REGEX), provider=st.sampled_from(["aws", "gcp", "k3d"])))
+@given(valid_components())
 def test_component(instance):
+    print(f"instance: {instance}")
     assert isinstance(instance.spec_version, int)
     assert isinstance(instance.spec_type, str)
     assert isinstance(instance.name, str)
     assert instance.name is not None
     assert instance.spec_version is not None
     assert instance.spec_type is not None
+    print("!!!!!!!!!!!!!!!!!!!!!!!!")
+    print("just prior to component type test")
     assert isinstance(instance.component_type, ComponentTypeEnum)
+    print("just after component type test")
+    print("!!!!!!!!!!!!!!!!!!!!!!!!")
```

Comment:
what are these? I'm guessing you used in testing? probably best to remove.

Reply:
Yea sorry. Good catch, haha.